### DISCUSSION TITLE
posts/33595/

### DISCUSSION THREAD

**Discussion by giscus** [STEVE-1936550490/blog_comment#16] [Announcements]
# posts/33595/

一部有点脑洞的纯爱小说改电影，讲的是一对时间流向相反的情侣，他们作为恋人的时间只有同在20岁时的短短30天。随着一方的回忆越来越多，另一边的记忆也会越来越少，对于男主的一个个第一次，对于女主来说却是最后一次了。
     “地球和月亮，从某种意义上说，也是在错过。相隔同样的距离，无法靠近，只有在短暂时间里可以看清对方。这如果是一对恋人，那该多虐啊。你们好歹能见到对方，见了面说开就好了啊。”

https://steve-1936550490.github.io/posts/33595/

**Comment by yzs020220** [comment]
给我干成泪人了，虽然我泪腺是真脆弱
